Makefile

```diff
@@ -527,7 +527,7 @@ test-full:
 test-full-386:
 	GOARCH=386 go test -coverprofile=coverage.txt -covermode=atomic ./lib/... ./app/...
 
-integration-test: victoria-metrics vmagent vmalert vmauth
+integration-test: victoria-metrics vmagent vmalert vmauth vmagent
```

Comment:
@makasim vmagent is already present in a targets list

Reply:
Ouh, I'll take care of it in the follow up PR.